Release packages (#1687)
iTwin/stratakit: packages/foundations/CHANGELOG.md

# Changelog

## 0.5.0

### Breaking changes

> - [#1565](https://github.com/iTwin/stratakit/pull/1565): Moved internal utilities from `@stratakit/foundations/secret-internals` subpath to `@stratakit/internal-utils` package.

Reviewer: ```suggestion
- [#1565](https://github.com/iTwin/stratakit/pull/1565): Removed internal APIs from `@stratakit/foundations/secret-internals`, moving them into the `@stratakit/internal-utils` package.
```

### Non-breaking changes

> - [#1497](https://github.com/iTwin/stratakit/pull/1497): Updated values for the following tokens:

Reviewer: I would explicitly list the dependency addition here.

```suggestion
- [#1565](https://github.com/iTwin/stratakit/pull/1565): Added a dependency on `@stratakit/internal-utils` package.
- [#1497](https://github.com/iTwin/stratakit/pull/1497): Updated values for the following tokens:
```

Reviewer 2: Good catch
